src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == repeatPassword && password == null) {
```

Comment:
let's simplify this check - use single if statement and think if we need to check both strings for null

Reply:
If i change it , test doesn't completed.In all cases , if i check 1 statement - test failed,if i did 1 if - test failed

Reply:
At Q&A Sonya said that this variant is correct

Reply:
No it's not, probably you misunderstood her. Your impl is overcomplicated now, we don't need to check password and repeat password for null  and for == and for equals, you're doing the same job several times